lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/sim/XwikiBridgeInterfaceRestResource.java

```diff
@@ -119,8 +119,28 @@ public void updateConfiguration(String coreFacadeHostname, int coreFacadeHostPor
 	
 	@Override
 	public ProcessData getProcessInfos(String processArtifactId) {
-		// TODO Auto-generated method stub
-		return null;
+		HttpClient httpClient = RestResource.getAnonymousClient();
+		String uri = String.format("%s/learnpad/sim/bridge/processes/%s",
+				RestResource.SIM_REST_URI,processArtifactId);
+
+		GetMethod getMethod = new GetMethod(uri);
+		getMethod.addRequestHeader("Content-Type", "application/json");
+		
+		ProcessData unmashelledProcessData = new ProcessData();
+		
+		try {
+
+			httpClient.executeMethod(getMethod);
+		    
+			ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
You don't have to create an ObjectMapper for each request. You can create a single (even static as long as you do not modify its configuration later!) one and reuse it.

Also, I assume this code may be called from multiple threads. ObjectMapper is thread safe so it should work fine, but for even better performance you can obtain an ObjectReader from the ObjectMapper which is supposed to be more lightweight and completely thread-safe.

You can refer to [this discussion](http://stackoverflow.com/questions/3907929/should-i-declare-jacksons-objectmapper-as-a-static-field) with an answer from the developer of Jackson himself.